.flake8

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2020 CERN
+# SPDX-License-Identifier: Apache-2.0
+
+# Minimal flake8 tweak to be compatible with black's line length
+# of 88 characters 
+# https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length
+
+[flake8]
```

Comment:
How is this enforced?

Reply:
It should be easy (we do it already in HSF/prmon)

- Add a cmake target `flake8` that runs the test <https://github.com/HSF/prmon/blob/575d3a42de48ea9f0a59ba8d018bba37dd9f8ab1/cmake/python-format.cmake#L16>
- Add an action script that invokes this test <https://github.com/HSF/prmon/blob/master/.github/scripts/flake8.sh>
- Add an action that triggers the script <https://github.com/HSF/prmon/blob/master/.github/workflows/flake8.yml>

The prmon action uses a container and we should probably have a project standard container environment for these kinds of checks.

Reply:
I'd guess by a CI check, though am not sure how to implement (but I know spack do this on their repo).